doc-content/enterprise-only/project-data/config-project-settings-proc.adoc

```diff
@@ -0,0 +1,69 @@
+[id='config-project-settings']
+
+= Configuring project settings
+
+Starting in {PRODUCT} {PRODUCT_VERSION}, {CENTRAL} contains additional project setting categories in the new process designer.
+
+.Prerequisites
+* You have created a {CENTRAL} project.
+
+.Procedure
+. To access the project *Settings* tab, in {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
+. Click on the project name.
+. Click *Settings* to view or modify the following project settings:
+
+* *General Settings* - Enables users to set a project's *Name*, *Description*, *Group ID*, *Artifact ID*, *Version*, and *Development Mode*. It also includes the following options:
```

Comment:
I try to avoid giving components the possessive: Consider:
to set the project .... attributes.

Reply:
Done